mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/ui/MerchantTransferActivity.java

```diff
@@ -137,14 +137,16 @@ public void onSlide(@NonNull View view, float v) {
     @OnClick(R.id.btn_submit)
     public void makeTransaction() {
         String externalId = tvMerchantVPA.getText().toString().trim();
-        String eamount = etAmount.getText().toString().trim();
+        String amount = etAmount.getText().toString().trim();
 
-        double amount = Double.parseDouble(eamount);
-        if (amount <= 0) {
+        if (amount.equals("")) {
+            showToast(Constants.PLEASE_ENTER_ALL_THE_FIELDS);
```

Comment:
For translation purposes, we have to add any strings in the string.xml folder. If you put the string inside the java classes then it is not possible to translate in other languages.

Reply:
I only used the strings which were already defined. Maybe propose this change in another PR

Reply:
Yeah, that will be great.